domain/src/test/java/org/oppia/domain/util/RatioExtensionsTest.kt

```diff
@@ -35,6 +35,12 @@ class RatioExtensionsTest {
     assertThat(ratioNonReduced.toSimplestForm()).isEqualTo(ratioReduced.ratioComponentList)
   }
 
+  @Test
+  fun testRatio_ratio_returnFormattedRatioString() {
```

Comment:
```suggestion
  fun testRatio_ratioWithThreeComponents_returnsFormattedRatioString() {
```

It's important to be unambiguous with test conditions.

Reply:
done